### PR TITLE
Today in Bitcoin/LN history snippet on homepage

### PR DESCRIPTION
**Pull request by urvishp80**
There two fields are as follows:

#### “random_flashback_posts”:
- selected random doc from the dataframe of all docs using df.sample(n=1)
- check if this selected entire thread has more than 5 replies
- if yes, add it to the list else randomly select another thread, max docs to collect = 2

#### “today_in_history_posts”:
- get today’s date and month
- fetch all the docs that have been published on this date-month
- check if it is an original post. if yes, add it to the list
- basically this will get only the original posts that were created on this date and not the replies.

### CODE REVIEW

File: static/homepage/Nov_2023/2023-11-20-homepage.json [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+                "David A. Harding",
+                "Jeremy",
+                "Johan Tor\u00e5s Halseth",
+                "Rusty Russell"
+            ],
+            "file_path": "static/lightning-dev/Nov_2018/001697_CPFP-Carve-Out-for-Fee-Prediction-Issues-in-Contracting-Applications-eg-Lightning-.xml",
+            "combined_summ_file_path": "static/lightning-dev/Nov_2018/combined_CPFP-Carve-Out-for-Fee-Prediction-Issues-in-Contracting-Applications-eg-Lightning-.xml"
+        }
+    ],
+    "today_in_history_posts": []
```
Comment (adamjonas) [inline static/homepage/Nov_2023/2023-11-20-homepage.json:288]: this remains empty. Can we just drop this and focus on one post for bitcoin and one for LN?
